packages/smithy-client/src/client.ts

```diff
@@ -8,6 +8,8 @@ import {
 
 export interface SmithyConfiguration<HandlerOptions> {
   requestHandler: RequestHandler<any, any, HandlerOptions>;
+  readonly apiVersion: string;
```

Comment:
Can you update this PR to just contain these changes, and also add documentation?